libs/blocks/merch/merch.js

```diff
@@ -32,15 +34,15 @@ export function getTacocatLocale(locale) {
   return { country, language };
 }
 
-export function getTacocatEnv(envName, locale) {
+export function getTacocatEnv(isProd = true, locale) {
```

Comment:
arguments with default values should be at the end in the list.
but do we still need env parameter to start with?

Reply:
discussed above, will change the order